compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -62,8 +62,8 @@ def __init__(
         search_authorizer: t.Any = None,
         fx_authorizer: t.Any = None,
         *,
-        code_serializer_method: SerializeBase | None = None,
-        data_serializer_method: SerializeBase | None = None,
+        code_serializer_strategy: SerializationStrategy | None = None,
+        data_serializer_strategy: SerializationStrategy | None = None,
```

Comment:
I think one of these should change for consistency:
`*_serializer_strategy` --> `*_serialization_strategy`
-- or --
`SerializationStrategy` --> `SerializerStrategy`